plugins/scanners/fossid/src/test/kotlin/FossIdTest.kt

```diff
@@ -935,6 +935,45 @@ class FossIdTest : WordSpec({
             }
         }
 
+        "apply exclusion rules to a origin scan" {
+            val projectCode = projectCode(PROJECT)
+            val scanCode = scanCode(PROJECT, FossId.DeltaTag.ORIGIN)
+            val config = createConfig()
```

Comment:
Your PR is adding the logic to convert rules in the case when this is NOT a delta scope.
Therefore you need a `createConfig(deltaScans= false)`.
You can test with the debugger that you are in the right branch of the code.

Reply:
Thanks for the hint. Using the debugger I am now sure that the desired code is covered.